bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckRun.java

```diff
@@ -723,6 +735,25 @@ private String getStatusMessage(PullRequestCheckIssueVisitor visitor,
                 }
                 progressBody.append("\n");
             }
+
+            // Update the issuePRMap
+            var prRecord = new PRRecord(pr.repository().name(), pr.id());
+
+            // Need previousIssues to delete associations
+            var previousIssues = BotUtils.parseIssues(pr.body());
+            // Add associations
+            for (String issueId : currentIssues) {
+                if (!previousIssues.contains(issueId)) {
+                    workItem.bot.addIssuePRMapping(issueId, prRecord);
+                }
+            }
+            // Delete associations
+            for (String oldIssueId : previousIssues) {
+                if (!currentIssues.contains(oldIssueId)) {
+                    workItem.bot.removeIssuePRMapping(oldIssueId, prRecord);
+                }
+            }
+            log.info("Map after updated: " + workItem.bot.issuePRMap());
```

Comment:
Logging the whole map here is probably not a good idea in production.

Reply:
Ok, I will remove it.